Edit Total Bills

timdoan22 opened the pull request:
Hey Kaleen I made the get bill total logic into an object to make it look cleaner.

Commit:
First commit

Commit:
Updated delete variable names

Commit:
Fixed markedComplete variable Names

Commit:
added total amount due

Commit:
added fontawesome icon

katiehom committed:
Added object to 'toLocaleString' of amountDue

I added the object to 'toLocaleString' in order to update decimals with only a tens place to add a zero and have two digits after the decimal.

katiehom committed:
Added object to 'toLocaleString' of billSum

Added the object to 'toLocaleString' of billSum within the 'res.render' of 'getBillTracker' to add two digits after a decimal point

Kaleen-MicheleXO committed:
Merge pull request #1 from katiehom/patch-1

Added object to 'toLocaleString' of amountDue

Kaleen-MicheleXO committed:
Merge pull request #2 from katiehom/patch-2

Added object to 'toLocaleString' of billSum

timdoan22 committed:
Added totalBills function for modularity and OOP standards

timdoan22 committed:
Fixed totalBills function to account for bills that are only marked as false

WonjaeSung committed:
Update billtracker.ejs

Due date now shows in MM/DD/YYYY format instead of YYYY-MM-DD

Kaleen-MicheleXO committed:
Merge pull request #5 from WonjaeSung/patch-3

Update billtracker.ejs

Kaleen-MicheleXO committed:
Merge pull request #3 from timdoan22/main

Billtrackers controller

timdoan22 committed:
Fixed comments related to totalBills function and commented out the billLeft variable

Kaleen-MicheleXO committed:
Merge pull request #6 from timdoan22/main

Fixed comments related to totalBills function and commented out the b…

Commit:
Updated styling and added materialize

Commit:
Updated styling and added materialize

Commit:
Updated styling and added materialize

Commit:
updated index.ejs styling added nav

kaitlinLewis committed:
Add files via upload

added logo

Kaleen-MicheleXO committed:
Merge pull request #7 from kaitlinLewis/main

Add files via upload

Commit:
styling

kaitlinLewis committed:
Add files via upload

small logo

Kaleen-MicheleXO committed:
Merge pull request #8 from kaitlinLewis/main

Add files via upload

Commit:
added logo updated styling

WonjaeSung committed:
made changes in css, and all the ejs files

WonjaeSung commented:
Added responsiveness to all pages. Fixed logo size and positioning issue.  Fixed table position issue. Logo is now linked to index.ejs.

Kaleen-MicheleXO committed:
Merge pull request #9 from WonjaeSung/main

made few changes on CSS, and ejs files.

katiehom committed:
Made style changes 

I switched from box-shadows to using Materialize classes, changed the style on the image on the index page and added a top margin, and I added a few rules to try to get the body and nav to stop shrinking.

katiehom committed:
Added classes and text below table

I added classes to form/buttons to add style and added text to instruct users to click on the text to change the status of a bill.

katiehom committed:
Added classes for styling

Added classes for the index image to add a top margin and Materialize box shadow

katiehom committed:
Added classes

Added classes for form and button styling

katiehom committed:
Added classes deleted unused code

I added classes for form and button styles and deleted our original form which was commented out.

Kaleen-MicheleXO committed:
Merge pull request #10 from katiehom/patch-3

Made style changes

Kaleen-MicheleXO committed:
Merge pull request #11 from katiehom/patch-4

Added classes and text below table

Kaleen-MicheleXO committed:
Merge pull request #12 from katiehom/patch-5

Added classes for styling

Kaleen-MicheleXO committed:
Merge pull request #13 from katiehom/patch-6

Added classes

Kaleen-MicheleXO committed:
Merge pull request #14 from katiehom/patch-7

Added classes deleted unused code

katiehom committed:
Missed one button

Added Materialize class to Signup button

katiehom committed:
Missed one button

Added Materialize class to the login button

katiehom committed:
Added styles to * and body

I added a box-sizing rule and set a 0 margin for the body hoping that it would help the resizing issue.

katiehom committed:
Added project gif

Added gif to ReadMe

Kaleen-MicheleXO committed:
Merge pull request #15 from katiehom/patch-8

Missed one button

Kaleen-MicheleXO committed:
Merge pull request #16 from katiehom/patch-9

Missed one button

Kaleen-MicheleXO committed:
Merge pull request #17 from katiehom/patch-10

Added styles to * and body

Kaleen-MicheleXO committed:
Merge pull request #18 from katiehom/patch-11

Added project gif

Jossyahia committed:
Update README.md

Jossyahia committed:
Update README.md

Jossyahia committed:
Update README.md

Kaleen-MicheleXO committed:
Merge pull request #19 from Jossyahia/main

added to the readme

Commit:
Updated media query

Commit:
updated package file to deploy

Kaleen-MicheleXO committed:
Update package.json

updated build

Kaleen-MicheleXO committed:
Update package.json

Kaleen-MicheleXO committed:
updated some packages

Kaleen-MicheleXO committed:
updated packages

Kaleen-MicheleXO committed:
Stop tracking .env file

Kaleen-MicheleXO committed:
config updates
